transcoder/src/info.go

```diff
@@ -297,13 +307,20 @@ func getInfo(path string) (*MediaInfo, error) {
 				Link:      link,
 			}
 		}),
-		Chapters: Map(make([]Chapter, max(chapters_end-chapters_begin, 1)-1), func(_ Chapter, i int) Chapter {
+		Chapters: Filter(Map(make([]Chapter, max(chapters_end-chapters_begin, 1)-1), func(_ Chapter, i int) Chapter {
+			startTime := mi.GetI(mediainfo.StreamMenu, 0, int(chapters_begin)+i, mediainfo.InfoName)
+			// We expect `startTime` to be something like `00:00:00`
```

Comment:
You probably forgot to remove that

Reply:
That was just a bit of documentation to explain why we do that. Let me know if you want me to delete it

Reply:
I was pointing to the printf but I guess github failed me